File: Hayaa.CompanyWebSecurity.Client/Model/AppFunction.cs
using Hayaa.BaseModel;
using System;
using System.Collections.Generic;
using System.Text;

namespace Hayaa.CompanyWebSecurity.Client.Model
{
    [Serializable]
    internal class AppFunction : BaseData
    {
        public int AppFuntionId { set; get; }
        public int AppId { set; get; }
        public String FunctionName { set; get; }
        public String ClassFullName { set; get; }
        public String InterfaceFullName { set; get; }
        public String Remark { set; get; }
        public String WebApiPath { set; get; }
        public byte Status { set; get; }
        public int AppServiceId { set; get; }

    }
}

File: v1/hayaa.scanservice.site/src/main/java/hayaa/scan/site/AppFunction.java
package hayaa.scan.site;

import hayaa.basemodel.model.BaseData;

public class AppFunction extends BaseData {
    private Integer appFuntionId;

    public Integer getAppFuntionId() {
        return appFuntionId;
    }

    public void setAppFuntionId(Integer appFuntionId) {
        this.appFuntionId = appFuntionId;
    }

    public String getFunctionName() {
        return functionName;
    }

    public void setFunctionName(String functionName) {
        this.functionName = functionName;
    }

    public String getPathName() {
        return pathName;
    }

    public void setPathName(String pathName) {
        this.pathName = pathName;
    }

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public byte getStatus() {
        return status;
    }

    public void setStatus(byte status) {
        this.status = status;
    }

    public Integer getAppServiceId() {
        return appServiceId;
    }

    public void setAppServiceId(Integer appServiceId) {
        this.appServiceId = appServiceId;
    }

    private String functionName;
    private String pathName;
    private String title;
    private byte status;
    private Integer appServiceId;
}
